wp-content/themes/core/assets/css/src/admin/master.pcss

```diff
@@ -14,9 +14,3 @@
 
 /* WP-Admin Variables */
 @import "_variables.pcss";
-
-/* panels */
-@import "panels/_conditionals.pcss";
```

Comment:
Dropped this partial b/c isn't no longer applicable. Gutenberg partials moved up tot he block-editor directory.

Reply:
👍